src/ray/core_worker/actor_handle.h

```diff
@@ -20,7 +20,7 @@
 #include "ray/common/task/task_util.h"
 #include "ray/core_worker/common.h"
 #include "ray/core_worker/context.h"
-#include "ray/gcs/redis_gcs_client.h"
+#include "ray/gcs/gcs_client/service_based_gcs_client.h"
```

Comment:
```suggestion
#include "src/ray/protobuf/gcs.pb.h"
```

Because actor_handle.h doesn't actually depend on GCS client.